pkg/networkmanager/firewall-client.js

```diff
@@ -81,6 +75,23 @@ function initFirewalldDbus() {
         firewall.enabledServices.delete(service);
         firewall.dispatchEvent('changed');
     });
+
+    firewalld_dbus.subscribe({
+        interface: 'org.fedoraproject.FirewallD1',
+        path: '/org/fedoraproject/FirewallD1',
+        member: 'Reloaded'
+    }, (path, iface, signal, args) => {
+        getServices();
+    });
+
+    getDefaultZonePath()
```

Comment:
I take it that shouldn't realistically fail in real life. However, could `getDefaultZonePath()` itself at least get a minimal `console.error` catch handler, so that it's easier to debug if this does fail somewhere?